.github/workflows/main.yml

```diff
@@ -62,6 +66,8 @@ jobs:
             RISC0_DEV_MODE: true
         run: forge test -vvv
 
+      - run: sccache --show-stats
+
```

Comment:
I don't think `sccache` is installed/initialized in this workflow right now. My thought on not including it is that we expect folks to clone/fork this repo and the likely host their code on GitHub. When they do, we'd like it to be simple for them to enable GitHub actions based CI. In that flow, `sccache` adds some friction as we've had to do quite a bit of work to get it running smoothly (but maybe I am overstating how much friction we've experienced)
```suggestion
```